server/game/cards/12-2T2D/PadreErnestodeDiaz.js

```diff
@@ -0,0 +1,24 @@
+const DudeCard = require('../../dudecard.js');
+
+class PadreErnestodeDiaz extends DudeCard {
+    setupCardAbilities(ability) {
+        this.reaction({
```

Comment:
Unfortunately, this is a `traitReaction` and as such, I am not sure how the `cost` work. If it does, good, but if not you will have to implement it similarly to for example `PhilipSwinford`.